.travis.yml

```diff
@@ -0,0 +1,17 @@
+sudo: false
+language: python
+env:
+  - PYTHON_VERSION=2.7
```

Comment:
Just to verify, we want py2? If so, then we won't be able to depend on skbio > 0.4.1, right?